client/src/router/routes.js

```diff
@@ -28,15 +25,16 @@ export const Routes = [
   {
     path: "/",
     name: "Home",
-    component: Home,
+
+    component: () => import("../views/Home.vue"),
```

Comment:
Was war dein Gedanke dahinter? Falls jemand direkt auf einen Link, z.B. zu einen Flug geht, soll Home nicht geladen werden?

Reply:
Ist eine rollup Optimierung. Die zu ladenden chunks werden kleiner. (Konnte man in stats.html gut sehen.)